tox.ini

```diff
@@ -1,12 +1,30 @@
 [tox]
-envlist =
-    py26,py27,py32,py33,py34
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
 
 [testenv]
-deps =
-    zope.testing
-    zope.testrunner
-setenv =
-    RUNNING_UNDER_TOX=1
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
 commands =
-    zope-testrunner --test-path=src
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=86
```

Comment:
I cannot verify that 86 is the correct number because `tox -e coverage` fails for me on this branch.

(I hate testing buildout extensions.  Always doctest failures due to irrelevant environmental details.)

Reply:
It is copied over from somewhere else and does not work here.
Let's wait for meta/config, too.

Reply:
This fails for me, because it thinks coverage is 40%.  Because it includes all of .tox/coverage/lib/python3.7/site-packages/ in the code it's measuring.

Where does the new metaconfig put coverage.py's configuration?  It needs a `src = z3c.recipe.tag`.

Reply:
No, I was hasty.  The coverage measured is in the low 30-ies because the test runs bin/buildout in a subprocess.

Reply:
It takes [just a little bit of configuration](https://github.com/buildout/buildout/pull/399/files#diff-1364cea14824861be52d9d5cc87b8c58R123) to make coverage and buildout tests play nice.

Reply:
I figured the magic necessary (by reading zc.buildout's documentation): #7.